### DISCUSSION TITLE
Making the disabled weekend example documentation clearer on the website

### DISCUSSION THREAD

**Discussion by tyeetale** [gpbl/react-day-picker#1839] [Ideas]
I'm new to the library and was introduced via Shadcn, but I had to dig for code examples and look into the types. Given that the type definitions are slightly different between v8 and v7, it might be best on the v8 website to provide simple example of disabling weekends for future noobs like me:

v7: 
```tsx
import React from 'react';
import DayPicker from 'react-day-picker';
import 'react-day-picker/lib/style.css';

export default function Example() {
  return (
    <DayPicker
      initialMonth={new Date(2017, 3)}
      disabledDays={[new Date(2017, 3, 12), { daysOfWeek: [0, 6] }]}
    />
  );
}
```

v8:
```tsx
import React from 'react';
import DayPicker from 'react-day-picker';

export default function Example() {
  return (
    <DayPicker
      disabled={{dayOfWeek: [0, 6]}}
    />
  );
}

**Comment by gpbl** [comment]
Hi @tyeetale! I'm sorry you couldn't find the info you needed in the docs.

Is the `dayOfWeek` vs `daysOfWeek` that you miss from the [upgrading from v7 guide](https://react-day-picker.js.org/guides/upgrading)?

**Reply by tyeetale** [reply]
@gpbl 

Hi! So the issue is that the v8 documentation just didn't show that you could disable the days. It only shows how to disable dates and dateranges. I'm just asking if it would be better to provide another example on the disabled section of the website, as I wouldn't be reading the upgrade from v7 guide at all if there was already a good example of how to disable weekends. 

I found the `dayOfWeek` by looking at the type definition of `Matcher` and found it there instead of `daysOfWeek`

**Comment by emrecoban** [comment]
You saved my life! I was thinking to create a new discussion about it. Thank @tyeetale!
